docs/DeveloperGuide.adoc

```diff
@@ -153,6 +153,73 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+=== Earnings Feature
+
+==== Add Earnings
+
+The `add_earnings` command allows for tutors to add their earnings into TutorAid.
+
+The format for the `add_earnings` command is as follows:
+
+ add_earnings d/<DATE> c/<CLASSID> amt/<AMOUNT>
+
+*Overview*
+
+The add claim `add_earnings` mechanism is facilitated by `AddEarningsCommand` and `AddEarningsCommandParser`, taking in the
+following input from the user: `Date`, `ClassId`, `Amount`, which will construct `Earnings` objects.
+
+.Add Earnings Command Sequence Diagram
+image::AddEarningsSequenceDiagram.png[]
```

Comment:
![image](https://user-images.githubusercontent.com/27663706/67536426-acf4c700-f709-11e9-9e83-ad047ca5d5f6.png)
Is your class still called AddressBookParser? Should it be renamed?